packages/compass-e2e-tests/helpers/commands/workspace-tabs.ts

```diff
@@ -48,17 +49,21 @@ export async function closeWorkspaceTabs(
     return (await browser.$$(Selectors.workspaceTab())).length;
   };
 
-  while ((await countTabs()) > 0) {
-    const currentActiveTab = await browser.$(
-      Selectors.workspaceTab({ active: true })
-    );
-    await currentActiveTab.click();
```

Comment:
Don't think this click was doing anything because the tab's already active.

Reply:
I think when I wrote it, the close button would only show up when the tab header was focused, but not when it's active, it might've been a bug that we fixed at some point

Reply:
I also changed this significantly a few days ago which is likely why it started flaking again and also I could have just missed this.